runtime/config-deployer-service/java/src/main/java/org/wso2/apk/config/DefinitionParserFactory.java

```diff
@@ -25,6 +25,8 @@ public static APIDefinition getParser(API api) {
             return new OAS3Parser();
         } else if (APIConstants.ParserType.ASYNC.name().equals(api.getType())) {
             return new AsyncApiParser();
+        } else if (APIConstants.ParserType.GRPC.name().equals(api.getType())) {
+            return new OAS3Parser();
```

Comment:
Can remove this else if block and add to the first if block:

if (APIConstants.ParserType.REST.name().equals(api.getType())
                || APIConstants.ParserType.GRAPHQL.name().equals(api.getType()) 
|| APIConstants.ParserType.GRPC.name().equals(api.getType()))